docs/sources/k6-studio/set-up/settings.md

```diff
@@ -12,9 +12,9 @@ k6 Studio includes a few settings you can configure by clicking on **Settings**
 
 ### Automatically detect browser
 
-When this setting is enabled, k6 Studio looks for Google Chrome in the default installation path in your system.
+When this setting is enabled, k6 Studio looks for Google Chrome or Chromium in the default installation path in your system.
```

Comment:
`Chromium` PR has not been merged yet and has been left out of this release, should we remove the mention here ? (only linux arm64 detects Chromium as of GA)

Reply:
I reworded it to:

```suggestion
When this setting is enabled, k6 Studio looks for Google Chrome in the default installation path in your system. For Linux ARM64 distributions, k6 Studio looks for Chromium instead.
```